GTG/plugins/send_email/sendEmail.py

```diff
@@ -32,22 +33,38 @@ class pluginSendEmail:
     """
     The plugin.
     """
+    def __init__(self):
+        """
+        Initialize variables.
+        """
+        self.browser_plugin_api = None
+        self.taskview_plugin_api = None
```

Comment:
We need these variables to store plugin api's, which are needed when removing the buttons on deactivating plugin